prowler/providers/aws/services/rds/rds_service.py

```diff
@@ -103,6 +103,13 @@ def _describe_db_instances(self, regional_client):
                                 ),
                                 port=instance.get("Endpoint", {}).get("Port"),
                                 vpc_id=instance.get("DBSubnetGroup", {}).get("VpcId"),
+                                subnet_ids=[
+                                    subnet_id["SubnetIdentifier"]
+                                    for subnet_id in instance.get(
+                                        "DBSubnetGroup", {}
+                                    ).get("Subnets", [])
+                                    if subnet_id["SubnetStatus"] == "Active"
```

Comment:
```suggestion
                                    if subnet_id.get("SubnetStatus") == "Active"
```
Do you think we should do this every time we access an array?"

Reply:
Those values are always set in the subnet.